ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/MultiTenancyPrivacyController.java

```diff
@@ -236,7 +236,8 @@ private void verifyPrivateFromMatchesEnclavePublicKey(
     }
   }
 
-  private void verifyPrivacyGroupContainsEnclavePublicKey(
+  @Override
+  public void verifyPrivacyGroupContainsEnclavePublicKey(
       final String privacyGroupId, final String enclavePublicKey) {
     final PrivacyGroup privacyGroup = enclave.retrievePrivacyGroup(privacyGroupId);
```

Comment:
are you sure that the privacyGroup cannot be null?

Reply:
Well, if it is null you won't find any matching privacy group, right?

And in the json-rpc method we are make the privacy group id a required parameter. I don't think we need to be extra defensive here in this internal class.